File: Android/Objects/src/com/apps/demo/designerscripts/LS_frmupdate.java
package com.apps.demo.designerscripts;
import anywheresoftware.b4a.objects.TextViewWrapper;
import anywheresoftware.b4a.objects.ImageViewWrapper;
import anywheresoftware.b4a.BA;


public class LS_frmupdate{

public static void LS_general(java.util.LinkedHashMap<String, anywheresoftware.b4a.keywords.LayoutBuilder.ViewWrapperAndAnchor> views, int width, int height, float scale) {
anywheresoftware.b4a.keywords.LayoutBuilder.setScaleRate(0.3);
anywheresoftware.b4a.keywords.LayoutBuilder.scaleAll(views);
views.get("panel1").vw.setLeft((int)((50d / 100 * width) - (views.get("panel1").vw.getWidth() / 2)));
views.get("pnlloading1").vw.setLeft((int)((50d / 100 * width) - (views.get("pnlloading1").vw.getWidth() / 2)));
views.get("pnlloading1").vw.setTop((int)((50d / 100 * width) - (views.get("pnlloading1").vw.getHeight() / 2)));
views.get("panel1").vw.setTop((int)((36d / 100 * height) - (views.get("panel1").vw.getHeight() / 2)));
views.get("imageview1").vw.setLeft((int)((views.get("panel1").vw.getWidth())/2d - (views.get("imageview1").vw.getWidth() / 2)));
views.get("btnlog").vw.setLeft((int)((views.get("panel1").vw.getWidth())/2d - (views.get("btnlog").vw.getWidth() / 2)));

}
}

File: iOS/Objects/src/B4iProject/LS_frmupdate.m

#import "iCore.h"
@interface LS_frmupdate:NSObject
@end

@implementation LS_frmupdate

- (void)LS_general:(B4ILayoutData*)views :(int)width :(int)height{
[B4ILayoutBuilder setScaleRate:0.3];
[B4ILayoutBuilder scaleAll:views :width :height];
[views get:@"panel1"].Left = (int)((50 / 100.0 * width) - ([views get:@"panel1"].Width / 2));
[views get:@"panel1"].Top = (int)((36 / 100.0 * height) - ([views get:@"panel1"].Height / 2));
[views get:@"imageview1"].Left = (int)(([views get:@"panel1"].Width)/2.0 - ([views get:@"imageview1"].Width / 2));

}
@end